server/src/test-utils/mocks/objects/events/mock-event-dto.ts

```diff
@@ -0,0 +1,21 @@
+import { EventDTO } from '../../../../modules/events/mappers/event-dto'
+
+const mockEventDTO = (id: string): EventDTO => {
+  return {
+    name: `Event Name ${id}`,
+    description: `Event Description ${id}`,
+    links: {
+      url: `Event URL ${id}`,
+      bannerImage: `Banner Image ${id}`,
+      facebook: `Facebook ${id}`,
+      twitter: `Twitter ${id}`,
+      instagram: `Instagram ${id}`,
+    },
+    startTime: '2021-01-01T00:00:00.000Z',
+    endTime: '2021-01-01T00:00:00.000Z',
```

Comment:
I wonder if the `endTime` being the same time as `startTime` will cause any problems in the future with testing?

Reply:
These new mock functions do the same thing as the old ones, just in a different file and with a different name. It's beyond the scope of this PR to actually change the tests' behaviour.

If it starts causing problems with testing in the future, the developer in the future can change it.